packages/firebase_ml/example/test_driver/firebase_model_manager.dart

```diff
@@ -7,17 +7,16 @@ part of 'firebase_ml.dart';
 void firebaseModelManagerTest() {
   group('$FirebaseModelManager', () {
     final FirebaseModelManager modelManager = FirebaseModelManager.instance;
-    final String MODEL_NAME = 'image_classification';
+    final String MODEL_NAME = 'myModelName';
+    final String INVALID_MODEL_NAME = 'invalidModelName';
 
-    test('downloadModel', () async {
-      FirebaseCustomRemoteModel model = FirebaseCustomRemoteModel(MODEL_NAME);
-      assert(model != null);
+    test('downloadModel and get its file', () async {
+      var model = FirebaseCustomRemoteModel(MODEL_NAME);
```

Comment:
If these are not changing can we use `final` here and in other places instead of `var`?